turbopack/crates/turbopack-core/src/source_map/utils.rs

```diff
@@ -84,7 +84,7 @@ pub async fn resolve_source_map_sources(
             .await?
         {
             let path_str = path.to_string().await?;
-            let source = format!("{SOURCE_MAP_PREFIX}{}", path_str);
+            let source = format!("{SOURCE_URL_PROTOCOL}{}", path_str);
```

Comment:
The three slashes are missing here, intentional?

Reply:
No. Do you know how to hit this branch? Seems like we don't have any tests for it.